utils/constants.js

```diff
@@ -28,6 +28,19 @@ const SKUS = {
   PICKUP_RJ_BARRA: '331',
 }
 
+const DOCUMENTS = {
```

Comment:
Why an object instead of an array?

Reply:
Just because I use it like `DOCUMENT[idx]` and I'm used to write this pattern with objects.